### PR TITLE
add @author tag in config and documenter

### PR DESCRIPTION
**Pull request by ifgx**
@joelday what do you think?

### CODE REVIEW

**Comment by ifgx** [comment]
@joelday what do you think?

File: src/documenter.ts [hunk 1]
```diff
@@ -215,6 +215,19 @@ export class Documenter implements vs.Disposable {
         sb.appendLine();
     }
 
+    private _emitAuthor(sb: utils.SnippetStringBuilder) {
+        if (vs.workspace.getConfiguration().get("docthis.includeAuthorTag", false)) {
+            let author: string = vs.workspace.getConfiguration().get("docthis.authorName", "-- SET AUTHOR VALUE IN CONFIGURATION docthis.authorDefault--");
```
Comment (joelday) [inline src/documenter.ts]: @ifgx WHY ARE YOU YELLING AT ME? ;)

File: src/documenter.ts [hunk 2]
```diff
@@ -110,7 +110,7 @@ export class Documenter implements vs.Disposable {
 
     private _printNodeInfo(node: ts.Node, sourceFile: ts.SourceFile) {
         const sb = new utils.StringBuilder();
-        sb.append(`${ node.getStart() } to ${ node.getEnd() } --- (${node.kind}) ${ (<any>ts).SyntaxKind[node.kind] }`);
+        sb.append(`${node.getStart()} to ${node.getEnd()} --- (${node.kind}) ${(<any>ts).SyntaxKind[node.kind]}`);
```
Comment (joelday) [inline src/documenter.ts]: No opinion about this whitespace convention, it just makes the diff hard to read.

File: src/documenter.ts (hunk 1 above)
Comment (joelday) [inline src/documenter.ts]: How about ```(Set the text for this tag by adding docthis.authorName to your settings file.)```

File: package.json [hunk 3]
```diff
@@ -85,6 +85,16 @@
...
           "description": "When enabled, will use names of params & methods as type hints."
+        },
+        "docthis.includeAuthorTag": {
+          "type": "boolean",
+          "default": false,
+          "description": "When enabled, will display the @author JSDoc tag, see config -> docthis.authorName ."
+        },
+        "docthis.authorName": {
+          "type": "string",
+          "default": "Firstname Name - <e-mail@address.com>",
```
Comment (joelday) [inline package.json]: Does this mean that https://github.com/joelday/vscode-docthis/pull/131/files#diff-09df5c5908fc1d7088c7834692eaf086R220 will never use the fallback text?

Comment (ifgx) [inline package.json]: Yes, it's not supposed to, but we never know... :D

File: src/documenter.ts (hunk 2 above)
Comment (ifgx) [inline src/documenter.ts]: Sorry about that, forgot about my plugin "format document on save". Not my intention.

File: src/documenter.ts (hunk 1 above)
Comment (ifgx) [inline src/documenter.ts]: @joelday as this text is not supposed to be displayed, if it is, then there is a problem. My intuition is to yell when there's a problem. But you're right, there's probably no reason for panicking here ^^

Comment (ifgx) [inline src/documenter.ts]: I like your proposition better.

**Comment by ifgx** [comment]
@joelday  I'm new to this GitHub PullRequest/edit thing. Is there anything I need to do to accept your requested changes, or is it your decision?

**Comment by joelday** [comment]
Sorry for the delay! I'll get it published a little later.

**Comment by joelday** [comment]
@ifgx Sorry, was groggy this morning. Please address those issues and make a new PR. Thanks!

File: src/documenter.ts (hunk 1 above)
Comment (joelday) [inline src/documenter.ts]: If the default value in package.json is used automatically, just copy this string into it and remove the default here.

File: package.json (hunk 3 above)
Comment (joelday) [inline package.json]: Let's just use this, but with the new string.

File: src/documenter.ts (hunk 2 above)
Comment (joelday) [inline src/documenter.ts]: @ifgx No problem, just revert it for consistency.

**Comment by ifgx** [comment]
@joelday yeah will do tomorrow morning UTC. :)

**Pull request merged**